src/main/helm/exec.ts

```diff
@@ -17,7 +17,7 @@ export async function execHelm(args: string[], options?: BaseEncodingOptions & E
   try {
     const opts = { ...options };
 
-    opts.env ??= process.env;
+    opts.env ??= { ...process.env };
```

Comment:
I think that our upcoming work should help make this less likely to slip though.